app/src/main/java/com/duckduckgo/app/tabs/ui/TabSwitcherActivity.kt

```diff
@@ -409,16 +425,30 @@ class TabSwitcherActivity : DuckDuckGoActivity(), TabSwitcherListener, Coroutine
                         )
                     }
                 }
+                TabSwitcherItem.TrackerAnimationTile -> Unit // TODO delete from list
             }
         }
     }
 
     override fun onTabMoved(from: Int, to: Int) {
-        val tabCount = viewModel.tabSwitcherItems.value?.size ?: 0
-        val canSwap = from in 0..< tabCount && to in 0..< tabCount
-        if (canSwap) {
-            tabsAdapter.onTabMoved(from, to)
-            viewModel.onTabMoved(from, to)
+        if (tabSwitcherAnimationFeature.self().isEnabled()) {
+            val isTrackerAnimationTileVisible = viewModel.tabSwitcherItems.value?.get(0) is TabSwitcherItem.TrackerAnimationTile
+            val canSwapFromIndex = if (isTrackerAnimationTileVisible) 1 else 0
+            val tabItemCount = viewModel.tabSwitcherItems.value?.count { it is TabSwitcherItem.Tab } ?: 0
```

Comment:
This fixes the last item issue:
```suggestion
            val tabItemCount = (viewModel.tabSwitcherItems.value?.count { it is TabSwitcherItem.Tab } ?: 0) + canSwapFromIndex
```

Reply:
@0nko I've made a fix for this, would be great if you could check please 🙏